views/home.htm

```diff
@@ -5,8 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="./home.css">
 </head>
 <body>
+    <h1>Leonardo is hot</h1>
     <a href="../views/home.htm"></a>
     <a href="../views/about.htm"></a>
     <a href="../views/works.htm"></a>
```

Comment:
- Be careful with the syntax! files should be named HTML - `about.html`, but u saved them as `about.htm`
- href is actually an URL link, not the name of your files - that's important to remember. YOu don't expect user to type `www.mywebsite.com/views/about.html` , we want only `www.mywebsite.com/about`
- Your `<a>` tags have no content (inner text), so they won't appear on the website right?
```
<a href="/about">About</a>
```